backend/capellacollab/users/tokens/routes.py

```diff
@@ -0,0 +1,59 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import fastapi
+from sqlalchemy import orm
+
+from capellacollab.core import database
+from capellacollab.core.authentication import injectables as auth_injectables
+from capellacollab.users import injectables as user_injectables
+from capellacollab.users import models as users_models
+
+from . import crud
+
+router = fastapi.APIRouter(
+    dependencies=[
+        fastapi.Depends(
+            auth_injectables.RoleVerification(
+                required_role=users_models.Role.USER
+            )
+        )
+    ]
+)
+
+
+@router.post("/current/token")
```

Comment:
```suggestion
@router.post("/current/tokens")
```

RESTful APIs usually use the plural form for resources: https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design